app/controllers/pockets_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PocketsController < AuthenticateController
   def index
-    query = Pocket.unclassified.where(organization_id: logged_user.organization.id).order('check_in desc')
+    query = Pocket.unclassified_and_checked_in.where(organization_id: logged_user.organization.id)
```

Comment:
Please add to be listed in descending order by check in

Reply:
+1

Reply:
I am a limado